src/runtime/image.ts

```diff
@@ -88,6 +89,26 @@ export function createImage (context, { providers, defaultProvider, presets }: C
     }
   })
 
+  image.lqip = async (source: string, options: any = {}) => {
+    const { src, provider: sourceProvider } = processSource(source)
+    const provider = providers[sourceProvider || options.provider || defaultProvider]
+    if (!provider) {
+      throw new Error('Unsupported provider ' + provider)
+    }
+
+    const generated = provider.provider.getImage(src, {
+      width: 30
+    }, provider.defaults)
+
+    if (typeof generated.getInfo !== 'function') {
```

Comment:
Maybe we can support `.getPlaceholder` to let providers have their own logic as well?